tests/functional/keys/full.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
```

Comment:
Several tests had
1. open `list`
2. go `full`
3. really test stuff

algorithms in `keys/tests.js`. With addition of this file, they are greatly simplified.